de/api/configuration-generate.md

```diff
@@ -86,11 +86,7 @@ We add routes for `/users/:id` in `nuxt.config.js`:
 ```js
 module.exports = {
   generate: {
-    routes: [
-      '/users/1',
-      '/users/2',
-      '/users/3'
-    ]
+    routes: ['/users/1', '/users/2', '/users/3']
```

Comment:
I don't think this makes it clearer

Reply:
It was Prettier that did this, actually. Seemed more than short enough not to split.
I split the french because it uses `/utilisateurs/` instead of `/users/` which imo is a bit much. Maybe the examples should be monolingual, so that if we later have a translation build process they can be imported from one place.